packages/mdl-ripple/animation.js

```diff
@@ -2,13 +2,31 @@
  * Create a sequential animation/transition from a list of class names.
  */
 export class SequentialAnimation {
+  /**
```

Comment:
I say we hold off on the closure jsdoc and `opt_` idioms until we decide to go full force with annotating v2 for closure. WDYT?